travis/Dockerfile-travis-web

```diff
@@ -13,5 +13,6 @@ WORKDIR /src
 COPY . /src
 
 RUN chown -R mitodl:mitodl /src
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
yea this line is unnecessary. the line above was necessary to avoid a permissions error.